gems/sorbet-runtime/test/types/utils.rb

```diff
@@ -69,5 +70,51 @@ def sigfun
         assert_equal('Integer', sfm.return_type.name)
       end
     end
+
+    describe 'T::Utils.unload_const' do
+      it 'releases const references from T::Private::Methods' do
```

Comment:
If in the future a new ivar in T::Private::Methods is used to store things, this test will not flag it. Would it be possible to use `ObjectSpace.reachable_objects_from` to be more robust?